qiskit/primitives/containers/estimator_pub.py

```diff
@@ -0,0 +1,168 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""
+Estimator Pub class
+"""
+
+from __future__ import annotations
+
+from typing import Tuple, Union
+from numbers import Real
+
+import numpy as np
+
+from qiskit import QuantumCircuit
+
+from .bindings_array import BindingsArray, BindingsArrayLike
+from .observables_array import ObservablesArray, ObservablesArrayLike
+from .shape import ShapedMixin
+
+
+class EstimatorPub(ShapedMixin):
+    """Primitive Unified Bloc for any Estimator primitive.
+
+    An estimator pub is essentially a tuple ``(circuit, observables, parameter_values, precision)``.
+
+    If precision is provided this should be used for the target precision of an
+    estimator, if ``precision=None`` the estimator will determine the target precision.
+    """
+
+    __slots__ = ("_circuit", "_observables", "_parameter_values", "_precision", "_shape")
+
+    def __init__(
+        self,
+        circuit: QuantumCircuit,
+        observables: ObservablesArray,
+        parameter_values: BindingsArray | None = None,
+        precision: float | None = None,
+        validate: bool = False,
```

Comment:
Shouldn't the default be `True`? I'd think most people prefer validation.

Reply:
Yeah I would agree we should probably default to True for validation.